pkgs/development/python-modules/sqlite-utils/default.nix

```diff
@@ -17,14 +17,16 @@
 
 buildPythonPackage rec {
   pname = "sqlite-utils";
-  version = "3.36";
+  version = "3.37";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-3MMROU/obcFvZQN7AHXiOO/P0uEuZdU+0ZaVRQKZbzw=";
+  src = fetchFromGitHub {
+    owner = "simonw";
+    repo = "${pname}";
```

Comment:
usage of `pname` is considered anti pattern https://github.com/NixOS/nixpkgs/issues/277994

Reply:
thanks @kirillrdy I've removed usage of `pname`